src/developer_config.yaml

```diff
@@ -3,16 +3,22 @@ global:
   logging_level: "DEBUG"
   table_config: "SingleLine"
   postcode_csv_check: False
+  network_or_hdfs: network #whether to load from hdfs or network (local Python)
 runlog_writer:
     write_csv: True # Write the runlog to a CSV file
     write_hdf5: False # Write the runlog to an HDF5 file
     write_sql: False # Write the runlog to a SQL database
     display: False # Display the runlog in the terminal
     log_path: "data/log"
-paths:
-  logs_foldername: "testing_pydoop"
+hdfs_paths:
+  logs_foldername: "resdev/logs/run_logs"
   snapshot_path: "/ons/rdbe_dev/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
-  masterlist_path: "data/external/ONSPD_NOV_2022_UK.csv"
+  postcode_masterlist: "data/external/ONSPD_NOV_2022_UK.csv"
+network_paths:
+  logs_foldername: "logs/run_logs"
```

Comment:
Directory has to be manually created. Maybe adding a line that creates it if it doesnt exist might help.